edit-post/components/header/more-menu/index.js

```diff
@@ -27,7 +28,13 @@ const MoreMenu = () => (
 		renderContent={ ( { onClose } ) => (
 			<div className="edit-post-more-menu__content">
 				<ModeSwitcher onSelect={ onClose } />
-				<FixedToolbarToggle onToggle={ onClose } />
+				<MenuGroup
+					label={ __( 'Settings' ) }
+					filterName="editPost.MoreMenu.settings"
+				>
+					<FixedToolbarToggle onToggle={ onClose } />
+					<TipsToggle onToggle={ onClose } />
+				</MenuGroup>
```

Comment:
This highlights that we have diverging interaction patterns within MoreMenu:
- Fix Toolbar to Top: click triggers action and closes menu
- Show Tips: _idem_
- Copy All Content: click triggers action, success is shown in item label, menu remains open

That said, for the purpose of this PR, we can move on. Just something to be aware of.